docs/topics/migrations.txt

```diff
@@ -561,7 +561,7 @@ the historical model and iterate over the rows::
         # version than this migration expects. We use the historical version.
         Person = apps.get_model('yourappname', 'Person')
         for person in Person.objects.all():
-            person.name = '%s %s' % (person.first_name, person.last_name)
+            person.name = f'{person.first_name} {person.last_name}'
```

Comment:
```suggestion
            person.name = f"{person.first_name} {person.last_name}"
```

Reply:
This is an example of a place I didn't want to change the quote style unless the quotes in the surrounding code were also changed.

I'd be happy to do a PR that changes the quotes, but that felt like a separate issue than f-string vs `%`.

Reply:
I agree with Trey, we shouldn't mix this up.

Reply:
Sorry, I was sure we applied black in all code example in in the documentation. If it's not I'll propose this final step in the black migration.

Reply:
Thanks to this PR I've created a new issue to propose to forma python code in the Django documentation
https://code.djangoproject.com/ticket/34140